docs/INTERFACE.md

```diff
@@ -0,0 +1,76 @@
+# Virtual machine Library
+The virtual machine library provides the following interfaces:
+
+` uintptr_t linux_setup_images(uintptr_t` *ram\_start*,
+                             uintptr_t *kernel*,
+                             size_t *kernel\_size*,
+                             uintptr_t *dtb\_src*,
+                             uintptr_t *dtb\_dest*,
+                             size_t *dtb\_size*,
+                             uintptr_t *initrd\_src*,
+                             uintptr_t *initrd\_dest*,
+                             size_t *initrd\_size*)`
+							 
+
+
+`linux_setup_images` is used to copy Linux kernel, initial RAM disc
+and device tree into guest RAM ready for execution.  The first
+argument *ram\_start* is the virtual address of the guest RAM.
```

Comment:
All of the function parameters should not be italics but in code blocks in my opinion, e.g `ram_start` instead of *ram\_start*

Reply:
I want to distinguish parameter names (that will be replaced with something) from symbols.  POSIX standard man pages use italic (or underline if italic isn't available).  That's why I used italic here.

Reply:
Ah okay fair enough then.